src/Execution/MutationExecutor.php

```diff
@@ -83,9 +83,9 @@ protected static function saveModelWithBelongsTo(Model $model, Collection $args,
             /** @var \Illuminate\Database\Eloquent\Relations\BelongsTo $relation */
             $relation = $model->{$relationName}();
 
-            collect($nestedOperations)->each(function ($values, string $operationKey) use ($relation, $model, $relationName): void {
+            Collection::make($nestedOperations)->each(function ($values, string $operationKey) use ($relation, $model, $relationName): void {
```

Comment:
Just noticed, can't we just use foreach here?

Reply:
We could, but I would prefer to do that in another PR

Reply:
Sure. I am finishing up nested updates tomorrow, will do it then.